src/core/tests/frontend/paddle/test_models/gen_scripts/generate_elementwise_ops.py

```diff
@@ -75,6 +75,28 @@ def elementwise_div(name : str, x, y, axis, in_dtype):
     return outs[0]
 
 
+def elementwise_mod(name : str, x, y, axis, in_dtype):
+    import paddle
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        node_x = paddle.static.data(name='x', shape=x.shape, dtype=in_dtype)
+        node_y = paddle.static.data(name='y', shape=y.shape, dtype=in_dtype)
+        out = paddle.fluid.layers.elementwise_mod(node_x, node_y, axis=axis)
```

Comment:
Paddle introduce a new api https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/floor_mod_en.html , which wrap the elementwise_mod. Can you add a testcase for this api? thanks.

Reply:
Please also rebase this branch on the latest master branch to make sure there is no conflicts with master.

Reply:
> Paddle introduce a new api https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/floor_mod_en.html , which wrap the elementwise_mod. Can you add a testcase for this api? thanks.

done.
![image](https://user-images.githubusercontent.com/72954905/191675810-0251c813-4c36-4a1d-9f54-d1d53b727a0d.png)
But the API `paddle.floor_mod` is supported sin paddle version 2.3, does the paddle version in ci environment match with 2.3?

Reply:
it's ok. it is also supported on v2.1